src/strategies/views/lights-view.test.ts

```
@@ -0,0 +1,9 @@
+import { describe, expect, test } from 'vitest';
+import { LightsViewStrategy } from './lights-view';
+
+describe('LightsViewStrategy.generate', () => {
+  test('returns empty lights view', async () => {
+    const view = await LightsViewStrategy.generate({}, {} as any);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)